src/main/java/com/denizenscript/ddiscordbot/objects/DiscordUserTag.java

```diff
@@ -240,6 +240,31 @@ public static void register() {
             return new ElementTag(object.getUser().isBot());
         });
 
+        // <--[tag]
+        // @attribute <DiscordUserTag.is_boosting[<group>]>
+        // @returns ElementTag(Boolean)
+        // @plugin dDiscordBot
+        // @description
+        // Returns a boolean indicating whether the user is boosting the specified group or not.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "is_boosting", (attribute, object) -> {

```

Comment:
could register with the required discordgrouptag input, see `<DiscordUserTag.is_in_group[<group>]>` for example

Reply:
Done 😊 Thank you